samples/jquery/00 ShoppingForm/src/modules/app/validation/formProductValidationService.js

```diff
@@ -0,0 +1,44 @@
+import {
+  FieldValidationResult,
+  Validators,
+  createFormValidation,
+} from 'lc-form-validation';
+import { nifValidator } from './rules/nifValidator';
+
+const DISCOUNT_REGEXP = /^[A-Z\d]{3}\-[A-Z\d]{4}[A-Z\d]{3}$/;
+
+const validationConstraints = {
+  fields: {
+    product: [
+      {
+        validator: Validators.required,
+        eventsFilter: { onSubmit: true, onChange: true },
```

Comment:
I think this line is not necessary because you are not using `onSubmit` eventFilter to filter one field.
In this [sample](https://github.com/Lemoncode/lcFormValidation/pull/60/files#diff-d7eb573276fa017c19ae6370d857b28aR90), you are using  `validateForm` to validate all fields at `onSubmit` event.

And  about `onChange` is the default filter, at this point as you want. If you want highlight that you can use eventsFilter, it's ok for me